### DISCUSSION TITLE
Jogosultság lekérdezés

### DISCUSSION THREAD

**Discussion by rrd108** [nav-gov-hu/M2M#55] [Ötletek]
Ha lenne egy endpoint amin ellenőrizni lehetne, hogy egy adott API kulccsal milyen cégekhez lehet hozzáférni, akkor ez kiderülne dokumentum beküldés előtt.

**Comment by TamasPill** [comment]
Arra vonatkozó szolgáltatást, hogy egy regisztrált felhasználó saját magán kívül milyen más szervezethez rendelkezik képviseleti joggal/meghatalmazással nem tervezünk készíteni. Az ilyen jellegű információkat a felhasználók saját maguknak tudják lekérdezni a NAV Ügyfélportálon keresztül.
